lib/cisco_node_utils/cmd_ref/itd_service.yaml

```diff
@@ -0,0 +1,120 @@
+# itd_service
+---
+_exclude: [N3k, N5k, N6k, N8k, ios_xr]
+
+_template:
+  get_command: "show running-config all | section itd"
+  get_context:
+      - '/^itd <name>/'
+  set_context: ["itd <name>"]
+
+access_list:
```

Comment:
How do you envision this working in the puppet manifest?  I feel like it's more intuitive to have `access_list => ''` rather then `access_list => false` to remove the access list.